### ISSUE TITLE
Symfony 5.2 + Easyadmin 3.2 + VichUploaderBundle 1.16

### ISSUE THREAD

**Issue by janongush** [dustin10/VichUploaderBundle#1175]
I'm running Symfony 5.2 + Easyadmin 3.2 + VichUploaderBundle 1.16 (used the outdated Documentation to set up) & a bit of help from this YT video https://www.youtube.com/watch?v=MS4LICZ1j0s

After following all the instructions & creating the requisite variables in the Object file, I'm getting the error message **"The "imageFile" image field must define the directory where the images are uploaded using the setUploadDir() method."**

So I went ahead and set up the setUploadDir method and here's an excerpt of the error message I got "An error has occurred resolving the options of the form "Vich\UploaderBundle\Form\Type\VichImageType": The options "upload_dir", "upload_filename" do not exist. "

A lot of the advice online has been along the lines of downgrading EasyAdmin but I don't think that's a valid solution. Help!

TL;DR: How can I make VichUploaderBundle 1.16 work with Symfony 5.2 + Easyadmin 3.2 in the absence of up to date documentation?

**Comment by garak** [comment]
This bundle does not use EasyAdminBundle.
EasyAdminBundle uses (or can use) this bundle.
So, you should ask integration instruction to EasyAdminBundle maintainers, not here.

**Comment by janongush** [comment]
Closing issue for now but to this wondering how to fix:

Instead of ImageField, use Field:

Like this:

Field::new('imageFile')->setFormType(VichImageType::class)->onlyOnDetail(),
ImageField::new('image')->setBasePath('/images'),
